docset/winserver2022-ps/activedirectory/Get-ADComputer.md

```diff
@@ -281,10 +339,10 @@ The following syntax uses Backus-Naur form to show how to use the Windows PowerS
 
 For a list of supported types for \<value\>, type `Get-Help about_ActiveDirectory_ObjectModel`.
 
-Note: Windows PowerShell wildcards other than \*, such as ?, are not supported by the *Filter* syntax.
-
-Note: To query using LDAP query strings, use the *LDAPFilter* parameter.
+Note: Windows PowerShell wildcards other than \*, such as ?, are not supported
+by the _Filter_ syntax.
```

Comment:
We have a special syntax for notes - we should probably use it while we're updating.

```suggestion
> [!NOTE]
> Wildcards other than `*`, such as `?`, are not supported by the **Filter** syntax.
```